resourcemanager/identity/system_and_user_assigned_map.go

```diff
@@ -11,10 +11,10 @@ import (
 var _ json.Marshaler = &SystemAndUserAssignedMap{}
 
 type SystemAndUserAssignedMap struct {
-	Type        Type                                   `json:"type"`
-	PrincipalId string                                 `json:"principalId"`
-	TenantId    string                                 `json:"tenantId"`
-	IdentityIds map[string]UserAssignedIdentityDetails `json:"userAssignedIdentities"`
+	Type        Type                                   `json:"type" tfschema:"type"`
+	PrincipalId string                                 `json:"principalId" tfschema:"principal_id"`
+	TenantId    string                                 `json:"tenantId" tfschema:"tenant_id"`
+	IdentityIds map[string]UserAssignedIdentityDetails `json:"userAssignedIdentities" tfschema:"identity_ids"`
```

Comment:
these are exposed in the schema as a list of ID's currently (the other metadata we may want to expose in the future, which is why it's plumbed through, fwiw) - so I don't think we can use this value directly right now?

Reply:
Ah ok, I guess we can take this one out for now until it's ready